src/http-util/src/lib.rs

```diff
@@ -38,17 +38,25 @@ where
 /// and two strings representing the (crate-local) paths to the production and development
 /// static files.
 macro_rules! make_handle_static {
-    ($static_dir:expr, $prod_base_path:expr, $dev_base_path:expr) => {
+    (
+        dir_1: $dir_1:expr,
+        $(dir_2: $dir_2:expr,)?
+        prod_base_path: $prod_base_path:expr,
+        dev_base_path: $dev_base_path:expr$(,)?
+    ) => {
         #[allow(clippy::unused_async)]
         pub async fn handle_static(
             path: ::axum::extract::Path<String>,
         ) -> impl ::axum::response::IntoResponse {
             #[cfg(not(feature = "dev-web"))]
-            const STATIC_DIR: ::include_dir::Dir = $static_dir;
+            const DIR_1: ::include_dir::Dir = $dir_1;
+            #[cfg(not(feature = "dev-web"))]
+            $(const DIR_2: ::include_dir::Dir = $dir_2;)?
```

Comment:
Should the cfg attribute go into the $(..) expansion?

Reply:
Good call, updated!